### PR TITLE
Updated the note to include current details on how to rename sites

### PR DESCRIPTION
**Pull request by modery**
The previous note (create a new site with desired URL and copy content) is not entirely correct anymore, as it is possible to rename site URLs now

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/change-your-sharepoint-domain-name.md [hunk 1]
```diff
@@ -34,6 +34,6 @@ Unfortunately, it isn't possible to change the SharePoint Online domain name for
 To use the domain name  *fourthcoffee.sharepoint.com*  , you would need to purchase a new Office 365 subscription and move all email, files, and any other data you want to keep to the new subscription. 
   
 > [!NOTE]
-> To change the name of a site, for example, from  *https://contoso.sharepoint.com/sites/sample1*  to  *https://contoso.sharepoint.com/sites/sample2*, you need to create a new site and then move all the content from the old site to the new one. For info about moving items in document libraries, see [Move or copy items from a SharePoint document library](https://support.office.com/article/00e2f483-4df3-46be-a861-1f5f0c1a87bc). 
+> To change the name of a site, for example, from  *https://contoso.sharepoint.com/sites/sample1*  to  *https://contoso.sharepoint.com/sites/sample2*, you need to access the modern SharePoint admin center from where this action can be performed on sites. See [SharePoint Online Can Now Rename Site URLs](https://techcommunity.microsoft.com/t5/Office-365/SharePoint-Online-Can-Now-Rename-Site-URLs/m-p/512293) for more details
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/change-your-sharepoint-domain-name.md]:
```suggestion
> To change the name of a site, for example, from  *https://contoso.sharepoint.com/sites/sample1*  to  *https://contoso.sharepoint.com/sites/sample2*, you need to access the modern SharePoint admin center from where this action can be performed on sites. See [SharePoint Online Can Now Rename Site URLs](https://techcommunity.microsoft.com/t5/Office-365/SharePoint-Online-Can-Now-Rename-Site-URLs/m-p/512293) for more details.
```

File: SharePoint/SharePointOnline/change-your-sharepoint-domain-name.md [hunk 2]
```diff
@@ -34,6 +34,6 @@ Unfortunately, it isn't possible to change the SharePoint Online domain name for
 To use the domain name  *fourthcoffee.sharepoint.com*  , you would need to purchase a new Office 365 subscription and move all email, files, and any other data you want to keep to the new subscription. 
   
 > [!NOTE]
-> To change the name of a site, for example, from  *https://contoso.sharepoint.com/sites/sample1*  to  *https://contoso.sharepoint.com/sites/sample2*, you need to create a new site and then move all the content from the old site to the new one. For info about moving items in document libraries, see [Move or copy items from a SharePoint document library](https://support.office.com/article/00e2f483-4df3-46be-a861-1f5f0c1a87bc). 
+> To change the name of a site, for example, from  *https://contoso.sharepoint.com/sites/sample1*  to  *https://contoso.sharepoint.com/sites/sample2*, you need to access the modern SharePoint admin center from where this action can be performed on sites. See [SharePoint Online Can Now Rename Site URLs](https://techcommunity.microsoft.com/t5/Office-365/SharePoint-Online-Can-Now-Rename-Site-URLs/m-p/512293) for more details.
```
Comment (nenonix) [inline SharePoint/SharePointOnline/change-your-sharepoint-domain-name.md]:
```suggestion
> To change the name of a site, for example, from  *https://contoso.sharepoint.com/sites/sample1*  to  *https://contoso.sharepoint.com/sites/sample2*, you need to access the modern SharePoint admin center. For more details, see [SharePoint Online Can Now Rename Site URLs](https://techcommunity.microsoft.com/t5/Office-365/SharePoint-Online-Can-Now-Rename-Site-URLs/m-p/512293).
```

File: SharePoint/SharePointOnline/change-your-sharepoint-domain-name.md [hunk 3]
```diff
@@ -34,6 +34,6 @@ Unfortunately, it isn't possible to change the SharePoint Online domain name for
 To use the domain name  *fourthcoffee.sharepoint.com*  , you would need to purchase a new Office 365 subscription and move all email, files, and any other data you want to keep to the new subscription.
```
Comment (nenonix) [inline SharePoint/SharePointOnline/change-your-sharepoint-domain-name.md]:
```suggestion
To use the domain name  *fourthcoffee.sharepoint.com*, you would need to purchase a new Office 365 subscription and move all email, files, and any other data you want to keep to the new subscription. 
```

**Comment by kaarins** [comment]
This is still rolling out. I will update when I publish the new article about this feature. :)

**Comment by scanum** [comment]
@kaarins Thanks Kaarin, please keep us updated. Appreciate all your help!

**Comment by scanum** [comment]
@kaarins Any updates? Thanks!

**Pull request merged**